packages/elements/src/components/ino-snackbar/ino-snackbar.scss

```diff
@@ -4,89 +4,125 @@
 @use 'base/theme';
 @use 'base/animation';
 
-$snackbar-types: 'error', 'warning';
+$snackbar-types: 'info', 'error', 'success';
+
+$snackbar-color-map-by-type: (
+  info: (
+    color: #2d5dff,
+    box-shadow-color: #2d5dff29,
+  ),
+  error: (
+    color: #7df481,
+    box-shadow-color: #7df48129,
+  ),
+  success: (
+    color: #f29191,
+    box-shadow-color: #f2919129,
+  ),
+);
```

Comment:
These colors are the new elements colors I guess. I think it is better to use the colors defined in `theme.color` or at least mention it in the issue that hanfles the implementation of the new color pallet.

Reply:
As we discussed earlier, I switched to the old color pallette for now.